src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -266,6 +271,13 @@ object ScalafixPlugin extends AutoPlugin {
   )
 
   override lazy val globalSettings: Seq[Def.Setting[_]] = Seq(
+    // use derive to support per-level values while allowing user to override at ThisBuild level
```

Comment:
the possible build-level override is `scalaVersion`, not `scalafixScalaBinaryVersion` as explained above